message MSG_DS_DC_RTABLE_POSITION_GET
{
    required uint32 dbID = 1;
    required string tableName = 2;
}

message MSG_DC_DS_RTABLE_POSITION_GET_ACK
{
    required uint32 dbID = 1;
    required string tableName = 2;
    optional string csIP = 3;
    required int32 statusCode = 4;
}
